easybuild/easyconfigs/r/rapidcsv/rapidcsv-8.64-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,35 @@
+# Contribution from Imperial College London/UK
+# uploaded: J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'rapidcsv'
+version = '8.64'
+
+homepage = 'https://github.com/d99kris/rapidcsv'
+description = """
+Rapidcsv is a C++ header-only library for CSV parsing. 
+While the name admittedly was inspired by the rapidjson project, 
+the objectives are not the same. The goal of rapidcsv is to be 
+an easy-to-use CSV library enabling rapid development. 
+For optimal performance (be it CPU or memory usage) a CSV parser 
+implemented for the specific use-case is likely to be more performant.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/d99kris/rapidcsv/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['e2ab5231b6e65f1e168dc279bbba2e34afd43c7bc6e2522726b107bcc4e8ebac']
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+    ('binutils', '2.38')
+]
+
+sanity_check_paths = {
+    'files': ['include/rapidcsv.h'],
+    'dirs': [''],
+}
+
+moduleclass = 'tools'
```

Comment:
@sassy-crick a small point perhaps, but I wonder whether this should be `data` rather than `tools`?

Reply:
Done. I have changed it here and in PR #15231 as well.